vtex-apps/b2b-quotes-graphql
# [Seller Quotes] feat: verify if seller accepts quotes to process splitting and notify

#### What problem is this solving?

Calls b2b-seller-quotes to verify if seller accepts quotes and to notify when a new quote is created.

#### How to test it?

- Clone branch `feat/verify-and-notify-quotes-endpoints` of [b2b-seller-quotes](https://github.com/vtex-apps/b2b-seller-quotes/tree/feat/verify-and-notify-quotes-endpoints) an execute `pnpm dev:vtex`
- Clone the branch of this PR, switch to bravtexfashionb2b account and link b2b-quotes-graphql on workspace `b2bsellerquoteslocalhost` (the same workspace name generated by `pnp:dev:vtex` on b2b-seller-quotes in the indicated branch of previous step)
- Create a quote that contains product of seller `shopfashion568`: https://b2bsellerquoteslocalhost--bravtexfashionb2b.myvtex.com/camisa-feminina-gray/p
- Observe the server logs of b2b-seller-quotes, where there should be 2 received requests:
  - `/b2b-seller-quotes/_v/0/verify-quote-settings`
  - `/b2b-seller-quotes/_v/0/notify-new-quote`



[Workspace](https://b2bsellerquoteslocalhost--bravtexfashionb2b.myvtex.com)
